apps/api/src/main.ts

```diff
@@ -28,20 +28,35 @@ async function initializeSentry() {
     logger.warn(
       'Missing one or more Sentry environment variables. Skipping initialization...'
     )
-  } else {
+    return
+  }
+
+  try {
+    const tracesSampleRate =
+      parseFloat(process.env.SENTRY_TRACES_SAMPLE_RATE) || 1.0
```

Comment:
This won't be needed if we are already parsing it in zod. Also, we can be certain that we wont need the `||` since if the config is not present, we don't allow the thing to boot up.

Reply:
removed this altogether as we handle this in the schema now